ui/src/components/ProvisionD-Config.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+    <TabMenu :model="TabMenuItem" :activeIndex="activeIndex"/>
+    <router-view></router-view>
+</template>
+<script setup lang="ts">
+
+import TabMenu from 'primevue/tabmenu';
+import { computed, ref } from 'vue';
+
+const activeIndex = ref(0);
+
+const TabMenuItem = ref([
```

Comment:
Can remove wrapping this in a ref() as this array will not need reactive updating

Reply:
Done